sdk/eventhub/event-hubs/src/eventDataBatch.ts

```diff
@@ -281,6 +286,7 @@ export class EventDataBatchImpl implements EventDataBatch {
    */
   public tryAdd(eventData: EventData, options: TryAddOptions = {}): boolean {
     throwTypeErrorIfParameterMissing(this._context.connectionId, "tryAdd", "eventData", eventData);
+    options = convertTryAddOptionsForCompatibility(options);
```

Comment:
The name felt like reading JAVA..

Reply:
Correct. It's ugly for a reason - I want to get rid of it and I want it to be obvious it's already overstayed its welcome. :)